### PR TITLE
UTF-8 unexpected behavior

### PR DESCRIPTION
**Pull request by chadbrewbaker**
Documenting unexpected behavior handling UTF-8.  Not only are the counts not the same as GNU 'wc', the pipe in fixture is returning all zero.

### CODE REVIEW

File: tests/by-util/test_wc.rs [hunk 1]
```diff
@@ -8,6 +8,17 @@ fn test_stdin_default() {
         .stdout_is("  13 109 772\n");
 }
 
+#[test]
```
Comment (sylvestre) [inline tests/by-util/test_wc.rs:11]: Could you run rustfmt on it ? thanks

Comment (chadbrewbaker) [reply tests/by-util/test_wc.rs:11]: Done.

**Comment by chadbrewbaker** [comment]
The MUSL i686 build seems to have flakey tests on chown.

```bash
failures:

---- test_chown::test_passgrp::test_gid2grp stdout ----
thread 'test_chown::test_passgrp::test_gid2grp' panicked at 'assertion failed: `(left == right)`
  left: `"root"`,
 right: `"�oot"`', /project/tests/by-util/test_chown.rs:38:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- test_chown::test_passgrp::test_uid2usr stdout ----
thread 'test_chown::test_passgrp::test_uid2usr' panicked at 'assertion failed: `(left == right)`
  left: `"root"`,
 right: `"x�\u{8}��\u{8}00"`', /project/tests/by-util/test_chown.rs:31:9
```

File: tests/by-util/test_wc.rs [hunk 2]
```diff
@@ -8,6 +8,16 @@ fn test_stdin_default() {
...
+fn test_utf8() {
+    new_ucmd!()
+        .args(&["-lwmcL"])
+        .pipe_in_fixture("UTF_8_test.txt")
+        .run()
+        .stdout_is(" 0 0 0 0 0\n");
+    //GNU returns "  300  2086 22219 22781    79"
```
Comment (sylvestre) [inline tests/by-util/test_wc.rs:18]:
```suggestion
    //GNU returns "  300  2086 22219 22781    79"
    // TODO: we should fix that to match GNU's behavior
```

**Pull request merged**